src/store/index.js

```diff
@@ -58,4 +68,32 @@ export function getThreads() {
   return threadObjects.sort((a, b) => b.rawTimestamp - a.rawTimestamp);
 }
 
+/* Object of possible mailbox names as defined by the gmail API, with values
+in my preferred format. Used to convert ugly mailbox names to pretty ones in
+getMailboxes() */
+const allMailboxes = {
```

Comment:
Usually global constants like this are named in `SCREAMING_SNAKE_CASE`. And what is this object actually? There could be other mailboxes, like if I tagged a message “Work.” Can you think of a name that more accurately captures the formatting/translating/making-human-readable mapping that this is encoding?

Additionally, since you’re falling back to the custom name whenever there’s not a key in this object, you could probably leave out all the keys you’re not changing at all and then it would be even more clear why this constant exists.

Reply:
I like your second point. It will keep it simpler to make it just an object of the mailbox names that need cleaning up. Thanks!